rasa/core/policies/flow_policy.py

```diff
@@ -370,23 +375,12 @@ def render_template_variables(text: str, context: Dict[Text, Any]) -> str:
         """Replace context variables in a text."""
         return Template(text).render(context)
 
-    def _slot_for_collect_information(self, collect_information: Text) -> Slot:
```

Comment:
Was this removed because it's not used anywhere anymore?

Reply:
yes, searched for it - isn't used anywhere.